ocs_ci/ocs/ocp.py

```diff
@@ -840,6 +840,50 @@ def get_logs(
         )
         return output
 
+    def get_build(self):
+        """
+        Return the OCP Build Version
+
+        Args:
+            None
+
+        Returns:
+             str: The build version of the OCP
+        """
+        ns = config.ENV_DATA['cluster_namespace']
+        out = run_cmd(f'oc get clusterversion -n {ns} -o yaml')
```

Comment:
plz avoid using run_cmd if there are better option (ocp.get etc.) - this apply for any place you are using run_cmd